nix/modules/common.nix

```diff
@@ -20,12 +20,12 @@ rec {
 
   daemonOptions = sharedOptions // {
 
-    baseProtocol = mkOption {
-      type = types.enum [ "013-PtJakart" "014-PtKathma" ];
+    baseProtocols = mkOption {
+      type = types.listOf (types.enum [ "013-PtJakart" "014-PtKathma" ]);
```

Comment:
I think there was a type for a list of unique elements, but im not sure

Reply:
there wasnt

Reply:
I haven't found such type as well:(